roles/job_runner/tasks/main.yml

```diff
@@ -8,70 +8,142 @@
     namespace: "{{ lookup('env', 'ANSIBLEJOB_NAMESPACE') }}"
   register: ansible_job
 
-- name: Launch a job, if error update AnsibleJob status then end play
+- name: Launch a job
   block:
-    - name: Launch job
-      awx.awx.tower_job_launch:
-        job_template: "{{ lookup('env','TOWER_JOB_TEMPLATE_NAME') }}"
-        extra_vars: "{{ ansible_job['resources'][0]['spec']['extra_vars'] | default(omit) }}"
-        inventory: "{{ ansible_job['resources'][0]['spec']['inventory'] | default(omit) }}"
-      register: job
-  rescue:
-    - name: Update status if job resulted in an error
-      k8s_status:
-        api_version: tower.ansible.com/v1alpha1
+  - name: Launch a job, if error update AnsibleJob status then end play
+    block:
```

Comment:
@rh-dluong since this has become a large block, what do you think about breaking this out into 2 different playbooks that get conditionally included with `include_tasks:`?

* launch_job.yml
* launch_workflow_job.yml

Reply:
Done